fapolicy_analyzer/ui/policy_rules_admin_page.py

```diff
@@ -324,29 +324,33 @@ def on_next_system(self, system):
         )
 
     def on_user_selection_changed(self, secondaryAction, data):
-        uid = data[1] if data else None
-        if uid != self.selectedUser:
-            self.selectedUser = uid
-            self.selectedGroup = None
-            self.__populate_acl_details(uid, acl.getUserDetails)
-            secondaryAction()
+        if data:
+            uids = [datum[1] for datum in data]
+        else:
+            uids = [None]
+        for uid in uids:
+            if uid != self.selectedUser:
+                self.selectedUser = uid
+                self.selectedGroup = None
+                self.__populate_acl_details(uid, acl.getUserDetails)
+                secondaryAction()
 
     def on_group_selection_changed(self, secondaryAction, data):
-        gid = data[1] if data else None
-        if gid != self.selectedGroup:
-            self.selectedGroup = gid
-            self.selectedUser = None
-            self.__populate_acl_details(gid, acl.getGroupDetails)
-            secondaryAction()
-
-    def on_subject_selection_changed(self, secondaryAction, data):
-        subject = data.file if data else None
-        if subject == self.selectedSubject:
+        gids = [datum[1] for datum in data] if data else [None]
+        for gid in gids:
+            if gid != self.selectedGroup:
+                self.selectedGroup = gid
+                self.selectedUser = None
+                self.__populate_acl_details(gid, acl.getGroupDetails)
+                secondaryAction()
+
+    def on_file_selection_changed(self, secondaryAction, data):
```

Comment:
I think keeping this as `on_subject_selection_changed` makes it more clear what it is used for.